File: app/src/main/java/com/shveed/cookmegood/fragment/CartFragment.java
package com.shveed.cookmegood.fragment;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import android.os.Bundle;
import androidx.recyclerview.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.shveed.cookmegood.adapter.CartRecipeAdapter;
import com.shveed.cookmegood.data.dto.Recipe;
import com.shveed.wallpapperparser.R;

import java.util.ArrayList;
import java.util.List;

public class CartFragment extends Fragment {

    private List<Recipe> recipes = new ArrayList<>();

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.f_cart, container, false);
        setRecipeData();
        RecyclerView recyclerView = (RecyclerView) view.findViewById(R.id.cartRecycler);
        CartRecipeAdapter adapter = new CartRecipeAdapter(recipes, getContext());
        recyclerView.setAdapter(adapter);
        return view;
    }


    private void setRecipeData(){
    }
}

File: app/src/main/java/com/shveed/cookmegood/fragment/CartFragment.kt
package com.shveed.cookmegood.fragment

import android.content.Intent
import android.view.View
import com.shveed.cookmegood.R
import com.shveed.cookmegood.activity.SuperActivity
import com.shveed.cookmegood.adapter.CartRecipeAdapter
import com.shveed.cookmegood.data.dto.Recipe
import kotlinx.android.synthetic.main.f_cart.*

class CartFragment: SuperFragment() {

    private val recipes: List<Recipe> = arrayListOf()

    private val cartAdapter: CartRecipeAdapter? = null

    override fun initInterface(view: View?) {
        (activity as SuperActivity).setSupportActionBar(cartFragmentToolbar)
        cartFragmentToolbar.title = getString(R.string.title_cart)
    }

    override fun setAttr() {
        setLayout(R.layout.f_cart)
    }

    override fun onResult(requestCode: Int, resultCode: Int, data: Intent?) {
    }
}